utils/build/docker/nodejs/uds-express4.Dockerfile

```diff
@@ -21,6 +21,7 @@ ENV PGDATABASE=system_tests
 ENV PGHOST=postgres
 ENV PGPORT=5433
 
+ENV DD_DATA_STREAMS_ENABLED=true
```

Comment:
I think you forgot to enable the env var for the typescript dockerfile no ? that's why the typescript tests are failing ?

Reply:
you are right :P thanks Simon